src/main/java/wenjigglybot/InvalidCommandException.java

```diff
@@ -0,0 +1,9 @@
+package wenjigglybot;
+
+public class InvalidCommandException extends Exception {
+    @Override
+    public String toString() {
```

Comment:
Maybe it is better to make use of getMessage() method instead of override toString() method?